src/components/HURUmap/Metrics/index.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import Metrics from "@/pesayetu/components/HURUmap/Metrics";
+
+export default {
+  title: "Components/HURUmap/Metrics",
+  argTypes: {},
+};
+
+const Template = ({ ...args }) => <Metrics {...args} />;
+
+export const Default = Template.bind({});
+
+Default.args = {
+  color: "Primary",
```

Comment:
@gertie-sheshe what other color options do we have? Tried to change color on storybook and color doesn't change

Reply:
There are [two](https://material-ui.com/api/linear-progress/): `Primary` and `Secondary @KhadijaMahanga Both match the colors used in the design